src/theme/metrics.ts

```diff
@@ -6,6 +6,8 @@ export const paragraphSpacing = 4;
 
 export const componentSpacing = 4;
 
+export const partnerSpacing = 10;
```

Comment:
do not add it here. it's not going to be used by other components. inline this variable. just do `spacing={10}` in index.tsx:120